src/core/css/inc/mixins.scss

```diff
@@ -5,3 +5,15 @@
   box-shadow:0 0 5px 0 $focus-outline-color;
   outline:none;
 }
+
+@mixin respond-to($breakpoint) {
+  @if map-has-key($breakpoints, $breakpoint) {
+    @media #{map-get($breakpoints, $breakpoint)} {
+      @content;
+    }
+  }
```

Comment:
yeah, maybe just something to think about if it get's bigger and more and more media queries are all over the place. Nothing that needs changing right now :)